components/exchange/Welcome.js

```diff
@@ -73,8 +73,13 @@ export default () => (
     <OnlyDesktop>
       <Title>Type:</Title>
       <ShortExplanation>
-        This indicates what type of word it is. Hover over the word type to
-        learn more.
+        This indicates what type of word it is.{' '}
+        <Link href="https://www.wordreference.com/fr/Abbreviations-French.aspx?dict=enfr">
+          <a href="https://www.wordreference.com/fr/Abbreviations-French.aspx?dict=enfr">
+            Click here to learn more
+          </a>
+        </Link>
```

Comment:
You only need href on Link. Like this:

```suggestion
        <Link href="https://www.wordreference.com/fr/Abbreviations-French.aspx?dict=enfr">
          <a>
            Click here to learn more
          </a>
        </Link>
```